skel/manifests/example-pod.yaml

```diff
@@ -1,10 +1,10 @@
 apiVersion: v1
 kind: Pod
 metadata:
-  name: alpine
+  name: example-pod
 spec:
   restartPolicy: Never
   containers:
-  - name: alpine
+  - name: example
   image: "alpine:3.2"
```

Comment:
`scratch` + command wouldn't work. To help prompt folks writing a chart, we may  as well leave this as something that does something, even if it is sleep.